modules/gdscript/gdscript_codegen.h

```diff
@@ -150,6 +150,8 @@ class GDScriptCodeGenerator {
 	virtual void write_end_jump_if_shared() = 0;
 	virtual void start_for(const GDScriptDataType &p_iterator_type, const GDScriptDataType &p_list_type) = 0;
 	virtual void write_for_assignment(const Address &p_list) = 0;
```

Comment:
```suggestion
	virtual void write_for_first_variable_assignment(const Address &p_list) = 0;
```

And in `gdscript_byte_codegen.h`.

Reply:
this method is related to the **for** container, not the first variable (and wasn't changed by me). changing to _write_for_container_assignment_ for code clarity